pkg/test/framework/components/istio/flags.go

```diff
@@ -45,4 +45,12 @@ func init() {
 		settingsFromCommandline.IngressGatewayServiceName,
 		`Specifies the name of the ingressgateway service to use when running tests in a preinstalled istio installation.
 		Should only be set when istio.test.kube.deploy=false`)
+	flag.StringVar(&settingsFromCommandline.IngressGatewayServiceNamespace, "istio.test.kube.ingressGatewayServiceNamespace",
```

Comment:
nit: we could allow `ingressGatewayServiceName=ns/name` format if its easier to have less flags. Not sure that is, though.

Reply:
Yeah, I'm not super opinionated here. I'll probably leave it as is unless you (or someone else) feels strongly that it should go the other way